File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_2.java
import java.util.Scanner;

public class exercicio_2 {

	public static void main(String[] args) {
		int num1, num2, soma, multiplicacao;
		Scanner teclado = new Scanner(System.in);

		System.out.print("Informe o primeiro numero: ");
		num1 = teclado.nextInt();
		System.out.print("Informe o segundo numero: ");
		num2 = teclado.nextInt();

		soma = num1 + num2;
		multiplicacao = soma * num1;

		System.out.println("O resultado da multiplicacao eh " + multiplicacao);
		
		teclado.close();
	}

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_2.c
#include <stdio.h>

int main() {
	int num1, num2, soma, multiplicacao;

	printf("Informe o primeiro numero: ");
	scanf("%d", &num1);

	printf("Informe o segundo numero: ");
	scanf("%d", &num2);

	soma = num1 + num2;
	multiplicacao = soma * num1;

	printf("O resultado da multiplicacao eh %d", multiplicacao);
}
